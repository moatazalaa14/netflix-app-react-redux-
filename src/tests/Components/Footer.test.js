import React from "react"
import ReactShallowRenderer from "react-test-renderer/shallow"
import Footer from "../../components/Layout/footer"



test("should Footer component rendered correctly",()=>{
    const renderer=new ReactShallowRenderer()
    expect(renderer.getRenderOutput).toMatchSnapshot()
})