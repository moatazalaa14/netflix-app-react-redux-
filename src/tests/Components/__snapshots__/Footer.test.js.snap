// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Footer component rendered correctly 1`] = `[Function]`;
